### DISCUSSION TITLE
Level Strip:  Click to swap drawing

### DISCUSSION THREAD

**Discussion by Mark11110** [opentoonz/opentoonz#6500] [Ideas]
Issue: Swapping drawing from the timeline is a bit unintuitive and time consuming. This becomes even more overwhelming if you've got lots of drawings in your level strip, you'd need to search the drawing number, click the timeline or xsheet, then enter the number.

Solution: an button in the level strip, so that we could swap drawings with just a click of a mouse.

side note: it would be cool if we could organize the xsheet by category and tag an alternate name on it - make search easy?

**Comment by gab3d** [comment]
"Drawing Substitution Backward/Forward" (Q / W) commands are what you're looking for? 🤔

**Comment by Mark11110** [comment]
It's close to what I'm looking for, but hopefully they'd still consider this request. :sweat_smile:
As I believe this modification can dramatically impacts productivity.

**Comment by DarrenTAnims** [comment]
"it would be cool if we could organize the xsheet by category and tag an alternate name on it - make search easy?"

What does this mean?
What categories?
And what are you searching for?

**Comment by artisteacher** [comment]
I've found Q/W works really well once you get used to it. I can see how it might not be ideal if there are a particularly large number of drawings and you want to switch to a drawing that's nowhere nearby in order.

If it's just a matter of having buttons, you can add the drawing substitution commands to the command bar.

I probably wouldn't add a button to the level strip. I could see adding an "Overwrite Current Cell" option in the Level Strip's context menu. The "Expose in Scene" option is helpful - but makes a new column/layer. Sometimes you just want to change or keep adding to the exposed frames in the current layer. This would be faster than drag/drop and give a visual alternative to Q/W for those cases where it helps to see the drawing you want to switch to.

Just an FYI, you can rename the columns to more easily identify them by double-clicking on the column names. It's not a searchable feature, but you could make your own "categories" that way. However if you need an option to search for your levels, you probably should try using sub-xsheets to keep your Xsheet more manageable.

**Comment by beeheemooth** [comment]
You can also use the Skeleton Tool in Animate mode to scroll drawings directly on the canvas (even if you are not using a skeleton in the scene).

**Comment by Mark11110** [comment]
> 
> 
> "it would be cool if we could organize the xsheet by category and tag an alternate name on it - make search easy?"
> 
> What does this mean?
> What categories?
> And what are you searching for?

I mean the level strip not xsheet
